### PR TITLE
new packages: ices, libshout (dep)

### PR DESCRIPTION
**Pull request by ErSoul**
Simple source client for an iceCast server.

### CODE REVIEW

File: packages/ices/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://icecast.org/ices/
+TERMUX_PKG_DESCRIPTION="IceS is a source client for the Icecast streaming server."
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.0.3
+TERMUX_PKG_REVISION=1
```
Comment (truboxl) [inline packages/ices/build.sh:6]:
```suggestion
```

File: packages/libshout/build.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://icecast.org/
+TERMUX_PKG_DESCRIPTION="Library which can be used to write a source client like ices."
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.4.6
+TERMUX_PKG_REVISION=1
```
Comment (truboxl) [inline packages/libshout/build.sh:6]:
```suggestion
```

**Review by truboxl — CHANGES_REQUESTED** [review]
`TERMUX_PKG_REVISION` is not needed. Also preferably add a newline at the end of each file.

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Please split the commit into two: I don't think "new package: ices" describe the changes done, since another new package, libshout is added as well.

Use one commit for each package

File: packages/ices/build.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://icecast.org/ices/
+TERMUX_PKG_DESCRIPTION="IceS is a source client for the Icecast streaming server."
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.0.3
+TERMUX_PKG_SRCURL=https://downloads.xiph.org/releases/ices/ices-$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=f710b2ea7ca39c4bdcfb6de61211e614b4504e35267619690799e29466c6537e
+TERMUX_PKG_DEPENDS="libshout"
+TERMUX_PKG_BUILD_IN_SRC=true
+
```
[the remark this answers is not in the document]
Comment (ErSoul) [reply packages/ices/build.sh:10]: Just added the two commits. Also renamed the PR...

**Comment by Grimler91** [comment]
Thanks!

**Comment by Grimler91** [comment]
Merged in https://github.com/termux/termux-packages/commit/7ec7ae519c2b02ada6df6362272aff165d96e724, but github's UI failed to detect it..

**Pull request closed without merging**